WordPress/gutenberg
# Babel Plugin Async Load: Lazy load react components when necessary

This PR tries to solve a blocker we have at the moment to ensure some of the packages are generic enough but at the same time allow lazy-loading some of their dependencies in the runtime code.

It does so by swapping the component with a lazy-loaded version at build time. Details on the README.

**Testing instructions**

 - Add the HTML block
 - Notice the editor component is lazy loaded.
